### PR TITLE
Release v1.2.8

### PR DESCRIPTION
**Pull request by bryanculver**
Request to merge `develop` into `main` for release v1.2.8.

Planned for 3/7.

**TODO:**
- [x] Update version in `pyproject.toml`
- [x] Update date in release notes

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request_beta.yaml [hunk 1]
```diff
@@ -0,0 +1,106 @@
+---
+name: ✨ Feature Request Beta
```
Comment (glennmatthews) [inline .github/ISSUE_TEMPLATE/feature_request_beta.yaml:2]: Are we going to change this to non-beta before the release?

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+Where our community participation happens in joint with another community or organization,
+for example the Network to Code Slack, vendor-hosted conferences, etc., we expect strict
+adherence with their community policies as well, as we are guests of their spaces.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+community@networktocode.com.
```
Comment (glennmatthews) [inline CODE_OF_CONDUCT.md:120]: Is this a live email address?

File: pyproject.toml [hunk 3]
```diff
@@ -2,7 +2,7 @@
 name = "nautobot"
 # Primary package version gets set here. This is used for publishing, and once
 # installed, `nautobot.__version__` will have this version number.
-version = "1.2.7"
+version = "1.2.8-beta.1"
```
Comment (glennmatthews) [inline pyproject.toml:5]: Reminder to update to "1.2.8" before merge.

File: nautobot/docs/release-notes/version-1.2.md [hunk 4]
```diff
@@ -140,6 +140,23 @@ All models that have `slug` fields now use `AutoSlugField` from the `django-exte
 
 Just as with the UI, the `slug` can still always be explicitly set if desired.
 
+## v1.2.8 (2022-MM-DD)
```
Comment (glennmatthews) [inline nautobot/docs/release-notes/version-1.2.md:143]: Reminder to add the date before merge.

File: .github/ISSUE_TEMPLATE/feature_request_beta.yaml (hunk 1 above)
Comment (bryanculver) [reply .github/ISSUE_TEMPLATE/feature_request_beta.yaml:2]: This is using the GH Issue Form method. Demoed with #1436. As long as everyone is OK with the slight representation differences, I will promote this before release. Needed to be merged to try it out here: https://github.com/nautobot/nautobot/issues/new?assignees=&labels=type%3A+feature&template=feature_request_beta.yaml

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (bryanculver) [reply CODE_OF_CONDUCT.md:120]: Added into the `nautobot-ansible` repo: https://github.com/nautobot/nautobot-ansible/blob/develop/CODE_OF_CONDUCT#L114

I assumed the powers at be did their due-diligence to set this up but it's worth verifying.

File: pyproject.toml (hunk 3 above)
Comment (bryanculver) [reply pyproject.toml:5]: 💡

File: nautobot/docs/release-notes/version-1.2.md (hunk 4 above)
Comment (bryanculver) [reply nautobot/docs/release-notes/version-1.2.md:143]: 💡

File: nautobot/docs/configuration/authentication/ldap.md [hunk 5]
```diff
@@ -74,27 +74,52 @@ AUTH_LDAP_CONNECTION_OPTIONS = {
...
+
+SSL can also be used by using the `ldaps://` URI scheme.
+
+```python
+AUTH_LDAP_SERVER_URI = "ldaps://ad.example.com"
+```
+
+#### Certificate Validation
+
+When using eithe RLS or SSL it is necessary to validate the certifiacte from your LDAP server.  Copy your CA cert to `/opt/nautobot/ca.pem`.
```
Comment (glennmatthews) [inline nautobot/docs/configuration/authentication/ldap.md:102]:
```suggestion
When using either TLS or SSL it is necessary to validate the certificate from your LDAP server.  Copy your CA cert to `/opt/nautobot/ca.pem`.
```

File: nautobot/docs/release-notes/version-1.2.md [hunk 6]
```diff
@@ -140,6 +140,28 @@ All models that have `slug` fields now use `AutoSlugField` from the `django-exte
...
+
+### Added
+
+- [#839](https://github.com/nautobot/nautobot/issues/839) - Add CODE_OF_CONDUCT.md to repository.
+- [#1299](https://github.com/nautobot/nautobot/pull/1299) - Add SECURITY.md to repository.
+- [#1388](https://github.com/nautobot/nautobot/pull/1388) - Added beta version of GitHub Issue Form style for feature request.
+- [#1419](https://github.com/nautobot/nautobot/pull/1419) - Add Documentation for Specifying CA Cert File for LDAP Auth Backend.
```
Comment (glennmatthews) [inline nautobot/docs/release-notes/version-1.2.md:150]:
```suggestion
- [#1419](https://github.com/nautobot/nautobot/pull/1419) - Add documentation for specifying a CA cert file for LDAP authentication backend.
```

**Review by jathanism — APPROVED** [review]

File: CODE_OF_CONDUCT.md [hunk 7]
```diff
@@ -0,0 +1,184 @@
+# Contributor Covenant Code of Conduct
```
Comment (glennmatthews) [inline CODE_OF_CONDUCT.md:1]: Should this say `Nautobot Community Code of Conduct` instead?

File: .github/ISSUE_TEMPLATE/feature_request_beta.yaml (hunk 1 above)
Comment (glennmatthews) [reply .github/ISSUE_TEMPLATE/feature_request_beta.yaml:2]: Looks good to me - I say go for it.

File: CODE_OF_CONDUCT.md (hunk 7 above)
Comment (jathanism) [reply CODE_OF_CONDUCT.md:1]: This is a community standard thing. https://www.contributor-covenant.org/version/2/1/code_of_conduct/

**Review by glennmatthews — APPROVED** [review]

**Pull request merged**